### PR TITLE
added feature block styles test

### PR DESCRIPTION
**Pull request by acostacos**
Desktop test does not pass due to the following issue: https://github.com/gambitph/Stackable/issues/1053

### CODE REVIEW

**Comment by blankart** [comment]
#### @acostacos TODOs Before Merging
- [ ] Pull master to this PR
- [ ] Register block snapshots for performance
- [ ] Introduce new adjust control function for adjusting alt texts
- [ ] Code review
- [ ] Passing tests

File: cypress/integration/v2/blocks/feature.spec.js [hunk 1]
```diff
@@ -109,6 +114,221 @@ function switchDesign() {
...
+		// ISSUE: Stretch Shape Mask updates in the back end but not in the front end
+		cy.adjust( 'Stretch Shape Mask', true ).assertClassName( 'img.ugb-img--shape', 'ugb-image--shape-stretch' )
+
+		// We won't be able to assert image size for now since it requires server handling.
+
+		cy.adjust( 'Alt Text (Alternative Text)', 'Hello World!' ).assertHtmlAttribute( '.ugb-img', 'alt', 'Hello World!' )
+	} )
+
+	cy.adjust( 'Image Width', 87, { viewport } )
+	cy.adjust( 'Force square image', true, { viewport } ).assertComputedStyle( {
```
Comment (legaspielaine) [inline cypress/integration/v2/blocks/feature.spec.js:185]: move this inside desktopOnly

File: cypress/integration/v2/blocks/feature.spec.js [hunk 2]
```diff
@@ -109,6 +114,221 @@ function switchDesign() {
...
+
+	desktopOnly( () => {
+		cy.adjust( 'Shape', {
+			label: 'Blob 1',
+			value: 'blob1',
+		} )
+		cy.adjust( 'Flip Shape Horizontally', true )
+		cy.adjust( 'Flip Shape Vertically', true )
+
+		// ISSUE: Stretch Shape Mask updates in the back end but not in the front end
```
Comment (blankart) [inline cypress/integration/v2/blocks/feature.spec.js:176]:
```suggestion

```

File: cypress/integration/v2/blocks/feature.spec.js (hunk 1 above)
Comment (acostacos) [reply cypress/integration/v2/blocks/feature.spec.js]: Got this!

File: cypress/integration/v2/blocks/feature.spec.js (hunk 2 above)
Comment (acostacos) [reply cypress/integration/v2/blocks/feature.spec.js]: Will remove the comment!

File: cypress/integration/v2/blocks/feature.spec.js [hunk 3]
```diff
@@ -109,6 +114,224 @@ function switchDesign() {
...
+		cy.adjust( 'Text Color', '#ffa03b' )
+		cy.adjust( 'Hover Effect', 'scale' ).assertClassName( '.ugb-button', 'ugb--hover-effect-scale' )
+		cy.adjust( 'Hover Opacity', 0.6 )
+		cy.adjust( 'Hover Colors', {
+			'Button Color #1': '#bd8b8b',
+			'Button Color #2': '#3fa35b',
+			'Gradient Direction (degrees)': 72,
+			'Text Color': '#80194d',
+		} )
+		cy.adjust( 'Typography', {
```
Comment (blankart) [inline cypress/integration/v2/blocks/feature.spec.js:259]: Let's use assertTypography here

File: cypress/integration/v2/blocks/feature.spec.js [hunk 4]
```diff
@@ -109,6 +114,224 @@ function switchDesign() {
...
+				'width': '41px',
+				'margin-right': '25px',
+			},
+		} )
+		cy.adjust( 'Adv. Icon Settings', {
+			'Icon Position': 'Right',
+		} ).assertClassName( '.ugb-button', 'ugb-button--icon-position-right' )
+	} )
+
+	cy.adjust( 'Typography', {
```
Comment (blankart) [inline cypress/integration/v2/blocks/feature.spec.js:304]: Let's use assertTypography here as well.

File: cypress/integration/v2/blocks/feature.spec.js (hunk 3 above)
Comment (acostacos) [reply cypress/integration/v2/blocks/feature.spec.js:259]: Got it will use that!

**Review by blankart — APPROVED** [review]

**Pull request merged**